swagger.yml

```diff
@@ -466,3 +482,23 @@ components:
         disabled:
           type: boolean
           example: false
+
+    StripeTransferRequest:
+      type: object
+      properties:
+        amount:
+          type: integer
+          example: 100
+        currency:
```

Comment:
Maybe replace for "asset", because in the whole project we are talking about asset and not currency.